tests/invalidrec.facts/methodLineNumber.facts

```diff
@@ -1,2 +1,4 @@
-click	/var/folders/5m/1vl49g8j5fq09m__0sy5js_c0000gn/T/gator-hw2s89d_/res//layout/activity_main.xml	83	com.example.invalidrec.MainActivity	-561734668
-click	/var/folders/5m/1vl49g8j5fq09m__0sy5js_c0000gn/T/gator-hw2s89d_/res//layout/activity_main.xml	112	com.example.invalidrec.MainActivity	1051323465
+&lt;com.example.invalidrec.MainActivity: void startRec(android.view.View)&gt;	method linked in java	83	com.example.invalidrec.MainActivity	256102585
+&lt;com.example.invalidrec.MainActivity: void stopRec(android.view.View)&gt;	method linked in java	112	com.example.invalidrec.MainActivity	-1685454288
```

Comment:
@LuxiWang99 Actually I am not sure if this is something you would expect. Please confirm this as well.

Reply:
It is a bug. I checked the test02 facts file and some methods that were linked via XML are now annotated as linked via Java.

Reply:
I added a commit to fix that.